Makefile

```diff
@@ -655,7 +655,7 @@ release-notes: $(RELEASE_NOTES_DIR) $(RELEASE_NOTES)
 
 .PHONY: promote-images
 promote-images: $(KPROMO)
-	$(KPROMO) pr --project cluster-api --tag $(RELEASE_TAG) --reviewers "$(IMAGE_REVIEWERS)" --fork $(USER_FORK)
+	$(KPROMO) pr --project cluster-api --tag $(RELEASE_TAG) --reviewers "$(IMAGE_REVIEWERS)" --fork $(USER_FORK) --image cluster-api-controller --image kubeadm-control-plane-controller --image kubeadm-bootstrap-controller --digests ""
```

Comment:
added `--digests ""` to workaround https://github.com/kubernetes-sigs/promo-tools/issues/546

Reply:
Should we either add a TODO here, open an issue or do we just subscribe to the upstream issue? (all fine for me)

Reply:
https://github.com/kubernetes-sigs/cluster-api/issues/6454